app/services/github_checks_verifier.rb

```diff
@@ -45,16 +46,20 @@ def filters_present?
     check_name.present? || check_regexp.present?
   end
 
+  def check_conclusion_allowed(check)
+    allowed_conclusions.include? check.conclusion
```

Comment:
@matiasalbarello do you know why it's saying this is Nil here?

Reply:
where? when running the workflow? or in some test?

Reply:
if it's when running the workflow, it's because you forgot to send some value in the `allowed-conclusions` param, therefore, it's sending nil and overwritting the default value